api/api/v3/cars/signals_list.md

```diff
@@ -65,6 +65,14 @@ All the *signals* you can access via the [signals API](signals.md)
 |`RearLeftDoorSts`|Represents the **Rear Left door** status |`0` = `closed `,<br /> `1` = `open`|
 |`RearRightDoorSts`|Represents the **Rear Right door** status |`0` = `closed`,<br /> `1` = `open`|
 |`TrunkSts`|Represents the **Trunk** status |`0` = `closed`,<br /> `1` = `open`|
+|**_Electrical Vehicle_**|||
+|`EVBatterySOC`|Represents the **Battery charge** state|Value is in `%`|
+|`EVBatteryVoltage`|Represents the **Battery voltage**|Value is in `V`|
+|`EVBatteryHealth`|Represents the **Battery health** state|Value is in `%`|
+|`EVWallPlugSts`|Represents the **Wall plug charger** status|`0` = `absent`,<br /> `1` = `present`|
```

Comment:
To match [dev.xee.com/signals](https://dev.xee.com/signals)

`0` = `Unplugged`
`1` = `Plugged`